src/Redux/api.tsx

```diff
@@ -766,11 +782,15 @@ const routes = {
     path: "/api/v1/test_sample/",
   },
   getTestSample: {
-    path: "/api/v1/test_sample",
+    path: "/api/v1/test_sample/{id}",
```

Comment:
@konavivekramakrishna ensure there's a trailing slash always, else it'd cause a redirect.
```suggestion
    path: "/api/v1/test_sample/{id}/",
```